src/main/java/org/openpnp/spi/base/AbstractPasteDispenseJobProcessor.java

```diff
@@ -0,0 +1,90 @@
+package org.openpnp.spi.base;
+
+import org.openpnp.model.Configuration;
+import org.openpnp.model.Part;
+import org.openpnp.spi.*;
+import org.openpnp.util.MovableUtils;
+
+public abstract class AbstractPasteDispenseJobProcessor extends AbstractJobProcessor
+        implements PasteDispenseJobProcessor {
+
```

Comment:
i'm thinking these funcs() can be removed - they are just left-over from when i took the abstractpnpjobprocessor class - not doing any harm, but may cause later confusion?